src/errors.rs

```diff
@@ -0,0 +1,8 @@
+//! Error handling.
+
+use thiserror::Error;
+
+/// Library errors
```

Comment:
The doc is too general.
Take a look at the std library <https://doc.rust-lang.org/src/core/str/mod.rs.html#144>

Reply:
I'm sorry. I'll fix it.